anypoint-mq/4.x/modules/ROOT/pages/anypoint-mq-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,77 @@
+= Upgrading and Migrating Anypoint MQ Connector Connector to 4.x 
+
+Upgrade Anypoint Connector for Anypoint MQ (Anypoint MQ Connector) to version 4.x.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|3.x |4.x
+|===
+
+== Changes in This Release
+
+Anypoint MQ Connector 4.x includes support for the Connected Apps feature.
+
+The changes included in this version break backward compatibility.
+
+This release contains a new format for the *URL* field in the *Global Element Properties* page.
+
+* In Anypoint MQ Connector 3.x, the URL format was:
++
+`https://mq-us-east-1.anypoint.mulesoft.com/api/v1`
+* In Anypoint MQ Connector 4.x, the URL format includes the orgID and envID. For example:
++
+`https://mq-us-east-1.anypoint.mulesoft.com/api/v1/organizations/orgID/environments/envId`
++
+image::amq-4x-publish-url.png[URL field in the Global Element Properties page]
++
+[IMPORTANT]
+====
+If you configure the connector with the wrong version of the URL,
+Anypoint MQ returns the following error:
+
+`Invalid api url provided`
+====
+
+== Upgrade Prerequisites
+
+Before you perform the upgrade, you must create a backup of your files, data, and configurations in case you need to revert to the previous version.
```

Comment:
We should probably ask them to identify the orgId and ENvId as well which are going to go in the new URL format? `https://mq-us-east-1.anypoint.mulesoft.com/api/v1/organizations/orgID/environments/envId`

Reply:
I added it as a step in the upgrade steps. I don't think it fits here.